api/admin.yaml

```diff
@@ -82,7 +82,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: 'https://raw.githubusercontent.com/moov-io/api/master/openapi-common.yaml#/components/schemas/Error'
+                $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
```

Comment:
This should be fixed so we can remove this change. 

https://github.com/moov-io/customers/pull/218

Reply:
Sweet, will rebase and update.